.github/helm/affine/values.yaml

```diff
@@ -36,14 +36,14 @@ graphql:
     type: ClusterIP
     port: 3000
     annotations:
-      cloud.google.com/backend-config: '{"default": "affine-backendconfig"}'
+      cloud.google.com/backend-config: '{"default": "affine-graphql-backendconfig"}'
```

Comment:
I suggest naming it `affine-api-backendconfig` rather than `affine-graphql-backendconfig` because it's a cross-application configuration.

Reply:
sync and gql server using different port now, so cannot share same config